### ISSUE TITLE
RetroArch x Beetle PSX HW Core x PlayStation 1 (MultiTap)

### ISSUE THREAD

**Issue by AlanRibas** [libretro/RetroArch#10759]
I have 4 xbox-one joysticks, connected to the computer via the wirelles receiver.
I am not able to connect all 4 joysticks to play “PlayStation 1” games that accept this amount of players. Like “Crash Bash” for example. 
I did not find the option to activate multitap. I don’t know if it’s necessary.

When I enter the configuration options of the controls, in a "playstation 1" game, I see the possibility of configuring a total of 5 joysticks.

When I enter the joysticks configuration menu, only the 1st and 2nd players have the option to choose the “playStation Controller” option in the “device type” (only these two josyticks work) . The other joystick configuration menus (3rd, 4th and 5th) only appear in that part the option “Retropad” (none of them work.).

In short. Is it possible or not to play playstation 1 games with 4 players using retroarch?

I’m using the core: Sony PlayStation (Beetle PSX HW).

Retroarch Version: 1.8.8

**Comment by bslenul** [comment]
You can, with Crash Bash:

* Start the game.
* Open RetroArch Quick Menu (F1 by default) and go to "Options".
* Find "Port 1: Multitap Enable" and turn it ON.
* Go to the very top of the same menu and click "Create Game Options file" to save that setting ONLY for that game (or else you'll have multitap enabled for every other games too).
* Go back to Quick Menu and click "Close Content" (clicking "Restart" won't work, you have to completely close the core).
* Start the game again and you should be able to play 4 players modes.

![Annotation 2020-06-01 114205](https://user-images.githubusercontent.com/33353403/83396875-f1adb280-a3fc-11ea-90d1-543218d52578.png)

**Comment by AlanRibas** [comment]
Thanks a lot for the help. And thank you very much also for detailing so well what I should do. I followed what I said and now it's perfect. I should have started using Retroarch years ago.

Thank you very much.

now it's time to have fun with friends in 4 player games.✌🏻😁✌🏻
                   🎮
